core/deployment/src/main/java/io/quarkus/deployment/configuration/BuildTimeConfigurationReader.java

```diff
@@ -1112,6 +1113,17 @@ private SmallRyeConfig getConfigForRuntimeRecording() {
             builder.getProfiles().add("");
             builder.getSources().clear();
             builder.getSourceProviders().clear();
+            builder.withCustomizers(new SmallRyeConfigBuilderCustomizer() {
```

Comment:
Could you clarify how the customizers could have registered `quarkus.log.file.path`?